tasks/src/zero-shot-classification/data.ts

```diff
@@ -0,0 +1,35 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Zero Shot Classification is the task of providing a large language model with a specific prompt requesting it to output a label or sequence of text that effectively answers or classifies the input",
```

Comment:
Can you look at here for inspiration? https://github.com/huggingface/hub-docs/blob/main/tasks/src/zero-shot-image-classification/data.ts 🙂